File: RebelsOpdrachten/src/app/models/item.ts
export type ItemResponse = {
    totalCount: number;
    items : Item[];
    lastEvaluatedKey: string;
}

export type Item = {
    uuid: string;
    titel: string;
    broker: string;
    locatie: string;
    uren: string;
    duur: string;
    link: string
}

File: RebelsOpdrachtenAPI/Models/Item.cs
using Amazon.DynamoDBv2.DataModel;

namespace RebelsOpdrachtenAPI.Models
{
    [DynamoDBTable("ICTRebelsOpdrachten")]
    public class Item
    {
        public string UUID { get; set; }
        public string Broker { get; set; }
        public string Titel { get; set; }
        public string Locatie { get; set; }
        public string Uren { get; set; }
        public string Duur { get; set; }
        public string Link { get; set; }
    }
}